pkg/local-gadget-manager/local-gadget-manager.go

```diff
@@ -275,11 +276,16 @@ func (l *LocalGadgetManager) Dump() string {
 }
 
 func NewManager() (*LocalGadgetManager, error) {
+	_, err := ebpf.RemoveMemlockRlimit()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
We can easily one-line these `if` if we only care about the `err`:
```suggestion
	if _, err := ebpf.RemoveMemlockRlimit(); err != nil {
		return nil, err
	}
```

Less modified lines :wink: 

With this approach, you don't even need to patch the lines after:
```diff
-	err := l.ContainerCollection.ContainerCollectionInitialize(
+	err = l.ContainerCollection.ContainerCollectionInitialize(
```

Reply:
Thanks. Updated